wwwapp/settings_prod.py

```diff
@@ -0,0 +1,32 @@
+from .settings_common import *
+
+SECRET_KEY = os.environ['OPENSHIFT_SECRET_TOKEN']
+
+DEBUG = False
+ALLOWED_HOSTS = ['warsztatywww.pl', 'www.warsztatywww.pl']
```

Comment:
`www.warsztatywww.pl` should be already handled by nginx redirect, is it really needed here?